src/tours/StartMenuTour.ts

```diff
@@ -8,76 +8,107 @@ export class StartMenuTour {
     return [
       {
         html: `<p>Welcome to this short tour providing an overview of the Ordino start menu!</p>
-
         <p>
           Use the "Next" button or the "Return" key to iterate through all the steps. You can use the
           <i>"Cancel"</i> button at any time to stop the tour and to interact with Ordino.
           Please note that the tour will load a new analysis session and the current
           one will be discarded.
-        </p>`,
+        </p>`
+      },
+
+      {
+        selector: 'ul[data-header="mainMenu"]',
```

Comment:
This point is missing: highlight all 3 tabs ("Datasets, Analysis Session, Tours")
The main menu is not highlighted for me

Reply:
Yes, the highlight is not so easy. I will see if I can find a solution.

Reply:
Fixed with b1ec4f1bb99480fa4a3df413106461929a6e0399.

![grafik](https://user-images.githubusercontent.com/5851088/122057362-d5ec3300-cdea-11eb-92a2-070aacc4ef22.png)